platforms/documentation/docs/src/snippets/buildCache/configure-built-in-caches/groovy/settings.gradle

```diff
@@ -4,6 +4,8 @@ rootProject.name = 'configure-built-in-caches'
 buildCache {
     local {
         directory = new File(rootDir, 'build-cache')
+        // enabled = false // Uncomment to disable the local build cache
+        // push = false // Uncomment to disable storing outputs in the build cache
```

Comment:
❓ I am not sure if `push` does anything for local cache.
I see that the property is exposed through the common interface.
But I wonder if you can have a "restore" only local cache setup.

@asodja Do you know?

Reply:
You can disable local push with `push = false`. What do you mean with `"restore" only local cache setup.`